pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -552,6 +552,13 @@ func (p *Provider) Create(
 	return g.Wait()
 }
 
+func min(a, b int) int {
```

Comment:
The duplication of primitive math functions is kinda annoying, thanks to Go's decision to provide only `math.Min(x, y float64)`. 

I was curious to see what other packages have done. There are several variants. E.g., this one does `a <= b` [1] which while isomorphic may behave differently under branch speculations.

I think it's okay to use `math.Min` as long as (a) integer args don't exceed float limits, (b) the code doesn't need to be performant. (Go's runtime will use optimized assembly, but `runtime.convT64` and special cases would add up.)
In any case, it's a nit :)

[1] https://github.com/cockroachdb/cockroach/blob/b0daa5f8d2af1c9c59ec79e09fb74a104c5bcbc2/pkg/sql/opt/props/cardinality.go#L154
[2] https://github.com/golang/go/blob/4f0d3bcd6d2d6bc726044153bf6875d195203b32/src/math/dim_amd64.s#L56

Reply:
I agree and searched the codebase to see how it was done elsewhere, and ultimately lifted the function.